scripts/spelling.txt

```diff
@@ -600,16 +684,20 @@ messsage||message
 messsages||messages
 microprocesspr||microprocessor
 milliseonds||milliseconds
```

Comment:
I've also seen use of "milli seconds" (with a space) that should be "milliseconds"  (micro too)